app-shells/mksh/mksh-54.recipe

```diff
@@ -20,7 +20,8 @@ SOURCE_URI="https://www.mirbsd.org/MirOS/dist/mir/mksh/mksh-R$portVersion.tgz"
 CHECKSUM_SHA256="8bce3837c386684aa7780f085f1a4dbd5e3e26fb5c528ee0d41ae29af7f39013"
 SOURCE_DIR="mksh"
 if [ "$targetArchitecture" != x86_gcc2 ]; then
-	PATCHES="mksh-$portVersion.patch"
+	PATCHES="mksh-$portVersion.patch
+				mksh-54.patch"
```

Comment:
This was OK because HaikuPorts sets portVersion to 54 when it is building mksh-54.recope.
So
```
PATCHES="mksh-$portVersion.patch"
```
is the same as
```
PATCHES="mksh-54.patch"
```
BTW, with this change the patch is referenced twice.
So you should just revert this tiny change.
Regarding the **`cmd:ed`** in BUILD_PREREQUIRES, it may be needed by TEST, so please make sure it is not required before you remove it from BUILD_PREREQUIRES.